src/shipit_static_analysis/shipit_static_analysis/report/mail.py

```diff
@@ -11,10 +11,21 @@
 logger = log.get_logger(__name__)
 
 
-EMAIL_HEADER = '''{nb_publishable} Publishable issues on Mozreview
+EMAIL_STATS_LINE = '|{source}|{total}|{publishable}|'
+
+EMAIL_HEADER = '''
+# Found {total} issues
+
+| Source | Issues | Publishable |
+|--------|--------|-------------|
+{stats_table}
+
+## Links
```

Comment:
Nit: I feel like this section title is not useful (the email layout is already very obvious, so this title is just extra noise).